config/locales/en.yml

```diff
@@ -1156,20 +1150,7 @@ en:
         other: "people"
     flash:
       not_using_wca: "This competition is not using WCA registration"
-      accepted_and_mailed:
-        one: "Registration accepted! Email notification sent."
-        other: "Registrations accepted! Email notifications sent."
-      rejected_and_mailed:
-        one: "Registration moved to waiting list"
-        other: "Registrations moved to waiting list"
-      deleted_and_mailed:
-        one: "Registration deleted"
-        other: "Registrations deleted"
```

Comment:
Do we have confirmation messages in the new system?

Reply:
Yes, but not including the and email notification sent bit